méli-mélo/2024-02-charts/js/charts.js

```diff
@@ -0,0 +1,165 @@
+/**
+ * @title WET-BOEW Charts plugin
+ * @overview Charts plugin based on Charts.js version 4.4.1
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author garneauma
+ */
+( function( $, window, wb ) {
+"use strict";
+/*
+	* Variable and function definitions.
+	* These are global to the plugin - meaning that they will be initialized once per page,
+	* not once per instance of plugin on the page. So, this is a good place to define
+	* variables that are common to all instances of the plugin on a page.
+	*/
+var componentName = "gc-chart",
```

Comment:
Not sure about that plugin name for now. It looks like to pretend this is going to be the GC branded chart but we didn't have those discussion yet.

What about to rely on using this data attribute `data-chart-type` which must be set on the table element for every chart? And the plugin "selector" can be: `[data-chart-type]`